www/data/Annoucement.ts

```diff
@@ -0,0 +1,11 @@
+type Annoucement = {
```

Comment:
there's really no formal structure to this website (sorry!) but we should probably keep type and data separate.

keep the announcement types in `/types` and change this back to a json file.

Reply:
Oh sorry I didn't notice, I will update that for sure!